notebook-controller/base/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+namePrefix: notebooks-controller-
+
+resources:
```

Comment:
is it common pattern to have one resource per file?

Reply:
Yeah, one set of resources per component